superdesk.config.js

```diff
@@ -7,6 +7,9 @@
 
 module.exports = function(grunt) {
     return {
+        apps: [
```

Comment:
It should be enabled from [main]( https://github.com/superdesk/superdesk-belga/blob/master/client/superdesk.config.js#L11) repositories instead of here

Reply:
sorry, forgot to exclude it